src/v2/schema.rs

```diff
@@ -50,15 +49,15 @@ pub struct Spec {
     pub definitions: Option<BTreeMap<String, Schema>>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub parameters: Option<BTreeMap<String, Parameter>>,
-    /// mappings to http response codes or "default"
+    /// Mappings to http response codes or "default"
     #[serde(skip_serializing_if = "Option::is_none")]
     pub responses: Option<BTreeMap<String, Response>>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub security_definitions: Option<BTreeMap<String, Security>>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub security: Option<Vec<BTreeMap<String, Vec<String>>>>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub external_docs: Option<Vec<ExternalDoc>>,
+    pub external_docs: Option<ExternalDoc>,
```

Comment:
The only fix that is relevant to match the spec.

Reply:
If so, how come e.g. title is no longer optional?